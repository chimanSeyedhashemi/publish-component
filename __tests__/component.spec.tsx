import * as React from "react";
import ExampleComponent from "..";
import {create} from "react-test-renderer";
 
test("Component should show 'red' text 'Hello World'", () => {
  const component = create(<ExampleComponent  text="World"/>);
  const testInstance = component.root;
 
  expect(testInstance.findByType(ExampleComponent).props.text).toBe("World");
 
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

