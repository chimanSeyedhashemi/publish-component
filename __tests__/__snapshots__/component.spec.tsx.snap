// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'red' text 'Hello World' 1`] = `
<div
  className="color-text well"
>
  Hello 
  World
</div>
`;
